front_end/src/actions/bookingActions.js

```diff
@@ -2,6 +2,7 @@ import {
   SET_BOOKING,
   SAVE_BOOKING,
   CLEAR_BOOKING,
+  CHANGE_RESERVATION,
   CANCEL_RESERVATION
 } from "./types";
 import axios from "axios";
```

Comment:
There's code that's been commented out in the action 'submitBooking'. Please remove if no longer needed. Otherwise leave a 'TODO:' comment if it depends on a future story.

Reply:
Good point. However, the commented code is from another story that I did not work on. It might be better to leave it there in case the author might need it.